Update 2 3 (#11)
Avaiga/taipy-getting-started-core: step_01/ReadMe.md

> You can download the code of this step [here](../src/step_01.py) or all the steps [here](https://github.com/Avaiga/taipy-getting-started-core/tree/develop/src).

> # Configuration and execution

Reviewer: What about something more appealing, like: 

```suggestion
# Playing with scenarios
```

> *Time to complete: 15min; Level: Beginner*

Reviewer 2: I think most style guides would ask for a space between the number and unit. I can't decide if "15 min." (note the period) or "15 mins" sounds better, so I would go with the full "15 minutes". Don't forget to change it for the other steps too if you agree with my suggestion!

```suggestion
*Time to complete: 15 minutes; Level: Beginner*
```

Before looking at some code examples, let’s define some basic terms Taipy Core uses. Taipy Core revolves around three major concepts.

## Three fundamental concepts in Taipy Core:

> - [**Data Nodes**](https://docs.taipy.io/en/latest/manuals/core/concepts/data-node/): are the translation of _variables_ in Taipy. Data Nodes know how to retrieve any type of data. They can refer to: any Python object (string, int, list, dict, model, data frame, etc.), a Pickle file, a CSV file, a SQL database, etc.

Reviewer: ```suggestion
- [**Data Node**](https://docs.taipy.io/en/latest/manuals/core/concepts/data-node/): is the translation of a _variable_ in Taipy. Data Nodes know how to retrieve any type of data. They can refer to: any Python object (string, int, list, dict, model, data frame, etc.), a Pickle file, a CSV file, a SQL database, etc.
```

> - [**Tasks**](https://docs.taipy.io/en/latest/manuals/core/concepts/task/): are the expression of _functions_ in Taipy.

Reviewer: ```suggestion
- [**Task**](https://docs.taipy.io/en/latest/manuals/core/concepts/task/): is the expression of a _function_ in Taipy.
```

> - [**Scenarios**](https://docs.taipy.io/en/latest/manuals/core/concepts/scenario/): are an instance of your pipelines. End-Users often require modifying various parameters to reflect different business situations. Taipy Scenarios provide the framework to "run"/"execute" pipelines under different conditions/variations (i.e., data/parameters modified by the end-user)

Reviewer: ```suggestion
- [**Scenario**](https://docs.taipy.io/en/latest/manuals/core/concepts/scenario/): is an instance of your pipelines. End-users often require modifying various parameters to reflect different business situations. Taipy Scenarios provide the framework to "run"/"execute" pipelines under different conditions/variations (i.e., data/parameters modified by the end-user).
```

- Or directly coding in Python.

> Let’s consider the simplest possible pipeline: a single function taking an input as a CSV dataset and generating a cleaned dataset. See below:

Reviewer: ```suggestion
Let’s consider the simplest possible DAG: a single function taking one input as a CSV dataset and generating a cleaned dataset. See below:
```

        ```python
        # Configuration of Data Nodes
        input_data_node_cfg = Config.configure_data_node("input_data",
                                                         storage_type="csv",
                                                         default_path="data.csv")
        clean_data_node_cfg = Config.configure_data_node("clean_data")

        # Configuration of tasks
        task_cfg = Config.configure_task("clean",
                                         clean,
                                         input_data_node_cfg,
                                         clean_data_node_cfg)

        # Configuration of the pipeline and scenario
        scenario_cfg = Config.configure_scenario_from_tasks("my_scenario", [task_cfg])
        ```

> The configurate is done! Let's use it to create scenarios and submit them.

Reviewer 2: typo

```suggestion
The configuration is done! Let's use it to create scenarios and submit them.
```

Value at the end of task 42
```    

## Gui-Core visual elements

Add these few lines to the code of your script. This creates a web application to:

- select scenarios you created,

- to create new ones,

- to submit them,

- to see the configuration used by the scenario.

```python
tp.Gui("""<|{scenario}|scenario_selector|>
          <|{scenario}|scenario|>
          <|{scenario}|scenario_dag|>""").run()
```

> [Put a gif here]

Reviewer 2: may have overlooked this!
